docs/writing-docs/doc-blocks.md

```diff
@@ -0,0 +1,370 @@
+---
+title: 'Doc Blocks'
+---
+
+Doc Blocks are the building blocks of Storybook documentation pages. By default, [DocsPage](./docs-page) uses a combination of the below blocks to build a page for each of your components automatically. 
+
+<div style="background-color:#F8FAFC">
+TODO: ask tom about both links below to where they should point to.
+</div>
+
+Also, custom [addons] can provide their own doc blocks, and there are [many available].
```

Comment:
Addons: https://docs.google.com/document/d/1BawdfqGhsP5tqhYQbVn29c6_rAz1ePmk9QayoXLDOKM/edit#heading=h.klqpukxeadji

many available => not yet published doc blocks blog post I think. (@domyen when is that going to be published?)

Reply:
Let's add this after 6.0 is released

Reply:
Removed the link, took a note of it and once Storybook 6 lands i'll update the docs accordingly.